src/evalWorker.js

```diff
@@ -30,6 +31,7 @@ function getMockApi () {
     BigInt,
     setTimeout,
     crypto,
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Rather than disabling this rule, we can change the empty functions to `() => undefined`, which will have the same effect. (I learned this after writing this package.)